onnxruntime/core/mlas/lib/qgemm_kernel_ummla.cpp

```diff
@@ -122,7 +122,7 @@ MlasGemmQuantCopyPackA<MLAS_GEMM_U8X8_KERNEL_UMMLA>(MLAS_GEMM_U8X8_KERNEL_UMMLA:
         const uint8_t* a6 = a0 + lda * 6;
         const uint8_t* a7 = a0 + lda * 7;
 
-        size_t k = CountK;
+        int k = static_cast<int>(CountK);
```

Comment:
same here. we don't need this change.

Reply:
this was the warning from android build I think. the file doesn't belong to sbgemm kernels. I will put a separate PR for this.